.github/workflows/argus-delete-stack-rdev.yaml

```diff
@@ -0,0 +1,20 @@
+name: Delete an RDev Stack
+
+on:
+  pull_request:
+    types: [ unlabeled, closed ]
+
+jobs:
+  delete-rdev-stack:
+    runs-on: [ARM64, self-hosted, Linux]
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'stack') || github.event.pull_request.state == 'closed' }}
+    permissions:
+        id-token: write
+        contents: read
+
+    steps:
```

Comment:
@hspitzley-czi can you change the BASE_URL to staging so we can register them in staging first?

Reply:
current default is staging

Reply:
when we deploy the api to prod i'll change the default

Reply:
👍🏻